### PR TITLE
Refresh exposure history screen

### PR DESCRIPTION
**Pull request by zhaccc**
**Why:**
Exposure history does not re-render unless we restart the app.

**What:**
Since the screens are mounted when the app starts and they newer unmount we relied on listeners to figure out when the app is in focused/blur state and caused re-render with the state change.
Also, we added pull to refresh functionality. To achieve that we compare previous exposure state with the current one to determine if we need to re-render the screen.

### CODE REVIEW

File: src/ExposureHistory/History/index.tsx [hunk 1]
```diff
@@ -34,11 +36,26 @@ const History: FunctionComponent<HistoryProps> = ({
...
+
+  useEffect(() => {
+    prevExposuresRef.current = exposures
+  })
 
   const handleOnPressMoreInfo = () => {
     navigation.navigate(Screens.MoreInfo)
   }
 
+  const handleRefresh = () => {
```
Comment (devinjameson) [inline src/ExposureHistory/History/index.tsx:50]: Thoughts on this for consistency?

```suggestion
  const handleOnRefresh = () => {
```

File: src/ExposureHistory/History/index.tsx [hunk 2]
```diff
@@ -51,6 +68,9 @@ const History: FunctionComponent<HistoryProps> = ({
           contentContainerStyle={style.contentContainer}
           style={style.container}
           alwaysBounceVertical={false}
+          refreshControl={
+            <RefreshControl refreshing={refreshing} onRefresh={handleRefresh} />
```
Comment (devinjameson) [inline src/ExposureHistory/History/index.tsx:72]:
```suggestion
            <RefreshControl refreshing={refreshing} onRefresh={handleOnRefresh} />
```

**Comment by devinjameson** [comment]
It may be worth checking with design if they'd like an icon at the top of this screen indicating that you can pull to refresh.

File: src/ExposureHistory/ExposureDetail.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ const ExposureDetail: FunctionComponent = () => {
     healthAuthorityAdviceUrl,
   } = useConfigurationContext()
 
-  const [connectivity, setConnectivity] = useState<boolean | null | undefined>(
+  const [isConnected, setIsConnected] = useState<boolean | null | undefined>(
```
[the remark this answers is not in the document]
Comment (zhaccc) [reply src/ExposureHistory/ExposureDetail.tsx:30]: Definitely but let's do a separate task for this. I'll cut a card and handle it.

**Comment by zhaccc** [comment]
> It may be worth checking with design if they'd like an icon at the top of this screen indicating that you can pull to refresh.

Good idea, I'll check with the team and we can cut a card for it!

**Pull request merged**